src/components/AgmExtension/AgmExtensionEvaluation.vue

```diff
@@ -10,7 +10,7 @@
 
     <template #content>
       <v-card
-        v-if="!data.isEligible"
+        v-if="evaluateResult && !data.isEligible"
```

Comment:
Added `evaluateResult` to only display the yellow message box, as well as the custom text in "Duration of Extension" when the extension request is valid (i.e. all fields for a specific scenario are entered)

Reply:
This partly resolves #18558. Want to take that ticket and fix it here?

Reply:
PS You don't have to.

Reply:
I don't mind fixing it while I'm already working in the code!